docs/developer/adding-community-components.md

```diff
@@ -0,0 +1,53 @@
+# Adding community components
+
+[Community components][cc] are components that are implemented and maintained by the community.
+
+## Before opening a proposal
```

Comment:
Open question: does "proposal" here mean the same thing as "proposal" in #909? 

If #909 gets accepted, I think we _should_ tie the two together, since proposals can be rejected for being technically infeasible, and that's one of the reasons we would not be able to accept a community component.